test/test-proxied-properties.js

```diff
@@ -181,4 +181,35 @@ describe("browser-polyfill", () => {
       });
     });
   });
+
+  describe("Privacy API", () => {
+    it("Should wrap chrome.privacy.* API", () => {
+      let lazyInitCount = 0;
+
+      const fakeChrome = {
+        privacy: {
+          get network() {
+            ++lazyInitCount;
+            const networkPredictionEnabled = {
+              get: () => {},
+              set: () => {},
+              clear: () => {},
+            };
+            return {networkPredictionEnabled};
+          },
+        },
+      };
+
+      return setupTestDOMWindow(fakeChrome).then(window => {
```

Comment:
`equal(lazyInitCount, 0, ...);` to check that `chrome.privacy` is not accessed at first.

Reply:
Done.